build.js

```diff
@@ -655,6 +655,27 @@ const buildIDLTests = (ast, globals) => {
       resources: resources
     });
 
+    const legacyWindowAlias = getExtAttr(iface, 'LegacyWindowAlias');
+    if (legacyWindowAlias) {
+      let aliases = legacyWindowAlias.rhs.value;
+      if (Array.isArray(aliases)) {
+        aliases = aliases.map((token) => token.value);
+      } else {
+        aliases = [aliases];
+      }
+      for (const alias of aliases) {
+        tests[`api.${alias}`] = compileTest({
```

Comment:
This doesn't quite work yet, because the api.SVGPoint test that gets created gets clobbered by another test that already exists: http://mdn-bcd-collector.appspot.com/tests/api/SVGPoint

And that custom test falls back to `createSVGPoint()`, suggesting that the test generated by this code here isn't really helpful.

Same story for everything except `webkitURL`. So generating these tests isn't the best approach here.